src/lib.rs

```diff
@@ -78,102 +75,114 @@ where
+    /// * `sensor_type` - Define whether a two, three or four wire sensor is
+    ///                   used.
+    /// * `filter_mode` - Specify the mains frequency that should be used to
+    ///                   filter out noise, e.g. 50Hz in Europe.
+    ///
     /// # Remarks
-    /// 
-    /// This will update the configuration register of the MAX31865 register. If the device doesn't properly react
-    /// to this, add a delay after calling `new` to increase the time that the chip select line is set high.
-    /// 
-    /// *Note*: The correct sensor configuration also requires changes to the PCB! Make sure to read the datasheet 
-    /// concerning this.
-    pub fn configure(&mut self, vbias: bool, conversion_mode: bool, one_shot: bool,
-        sensor_type: SensorType, filter_mode: FilterMode) -> Result<(), E> {
-        let conf : u8 = ((vbias as u8) << 7) |
-            ((conversion_mode as u8) << 6) |
-            ((one_shot as u8) << 5) |
-            ((sensor_type as u8) << 4) | 
-            (filter_mode as u8);
+    ///
+    /// This will update the configuration register of the MAX31865 register. If
+    /// the device doesn't properly react to this, add a delay after calling
+    /// `new` to increase the time that the chip select line is set high.
+    ///
+    /// *Note*: The correct sensor configuration also requires changes to the
+    /// PCB! Make sure to read the data sheet concerning this.
+    pub fn configure( 
+        &mut self,
+        vbias: bool,
+        conversion_mode: bool,
+        one_shot: bool,
+        sensor_type: SensorType,
+        filter_mode: FilterMode,
+    ) -> Result<(), E> {
+        let conf: u8 = ((vbias as u8) << 7)
+            | ((conversion_mode as u8) << 6)
+            | ((one_shot as u8) << 5)
+            | ((sensor_type as u8) << 4)
+            | (filter_mode as u8);
 
         self.write(Register::CONFIG, conf)?;
 
         Ok(())
     }
 
-    /// Set the calibration reference resistance.
-    /// This can be used to calibrate inaccuracies of both the reference resistor 
-    /// and the PT100 element.
-    /// 
+    /// Set the calibration reference resistance. This can be used to calibrate
+    /// inaccuracies of both the reference resistor and the PT100 element.
+    ///
     /// # Arguments
-    /// 
+    ///
     /// * `calib` - A 32 bit integer specifying the reference resistance in ohms
     ///             multiplied by 100, e.g. `40000` for 400 Ohms
-    /// 
+    ///
     /// # Remarks
-    /// 
-    /// You can perform calibration by putting the sensor in boiling (100 degrees 
-    /// Celcius) water and then measuring the raw value using `read_raw`. Calculate 
-    /// `calib` as `(13851 << 15) / raw >> 1`.
-    pub fn set_calibration(&mut self, calib : u32) -> Result<(), E> {
+    ///
+    /// You can perform calibration by putting the sensor in boiling (100
+    /// degrees Celcius) water and then measuring the raw value using
```

Comment:
I think Celcius is a recurring typo, it should probably be Celsius with an S?

Reply:
Indeed. I think I have now fixed all occurrences of this.